packages/presets/src/account.cairo

```diff
@@ -27,13 +27,13 @@ pub mod AccountUpgradeable {
     impl UpgradeableInternalImpl = UpgradeableComponent::InternalImpl<ContractState>;
 
     #[storage]
-    struct Storage {
+    pub struct Storage {
         #[substorage(v0)]
-        account: AccountComponent::Storage,
+        pub account: AccountComponent::Storage,
         #[substorage(v0)]
-        src5: SRC5Component::Storage,
+        pub src5: SRC5Component::Storage,
         #[substorage(v0)]
-        upgradeable: UpgradeableComponent::Storage
+        pub upgradeable: UpgradeableComponent::Storage
```

Comment:
Since presets are not meant to be customizable, should we remove the pubs? Tests should only be using the dispatchers

Reply:
Actually, the erc721 preset is an exception because we're calling erc721 methods in the `mint_assets` test. I think we could test `mint_assets` as a side effect in the constructor tests if we want to remove the pubs

Reply:
I remember we had this conversation during our weekly call, and decided that for consistency(today all storage structs and it's members are pub) we would keep it that way, but want to start taking a more opinionated approach in future releases. Are there any presets that you think we should already make their structs(or some members) private?

Reply:
Ahh we did, forgive me 😅 there aren't any presets in particular. I get the idea to keep them public for consistency; however, the class hash tends to change anyway when we bump versions. Since we did make the plan to start taking an opinionated approach in the future, I won't block this PR on this